applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -22,11 +23,23 @@
          ,delete/2
         ]).
 
--include("crossbar.hrl").
+-include_lib("crossbar/src/crossbar.hrl").
```

Comment:
then do it in a pr not this one. what you're proposing needs to be debated with the reswt of the team. maybe sync is not the right tool or needs modifications.